Demo/Fullscreen/FullscreenGallery/FullscreenGalleryDemoViewController.swift

```diff
@@ -0,0 +1,304 @@
+//
+// Copyright (c) 2019 FINN AS. All rights reserved.
+//
+
+import FinniversKit
+
+// MARK: - Helpers
+
+private struct DemoViewModel: FullscreenGalleryViewModel {
+    let imageUrls: [String]
+    let imageCaptions: [String]
+    let selectedIndex: Int
+}
+
+private class DemoPreviewCell: UICollectionViewCell {
+    private(set) var imageIndex: Int?
+    private(set) var imageUrl: String?
+
+    private(set) lazy var imageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.contentMode = .scaleAspectFit
+        return imageView
+    }()
+
+    // MARK: - Init
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setup()
+    }
+
+    private func setup() {
+        contentView.addSubview(imageView)
+        imageView.fillInSuperview(insets: UIEdgeInsets(all: .mediumSpacing))
```

Comment:
Worth noting that `fillInSuperview(insets:)` adds constraints directly from the values from `UIEdgeInsets` to the corresponding anchors. I.e. it does not add a minus `-` to values for `trailingAnchor` or `bottomAnchor`, where you have set the value `.mediumSpacing`, aka. 8 points.

I guess what you really want is `-.mediumSpacing`, aka. -8 points.

Reply:
Hmmm, nice spot – is the issue in `fillInSuperview(insets:,isActive:)`? I would expect `UIEdgeInsets(leading: x, top: x, trailing: x, bottom: x)` to be equivalent to `x` pixels of padding, and the bottom & trailing constraints thus having `constant: -x`.

Reply:
Yup, I agree. I've run into the same issue, and when I saw you added `init(all:)` I had to check the documentation.

For `UIEdgeInsets` it says that positive values inset towards the center, while negative values expand from the center. This is not the case for `fillInSuperview(insets:)`, where it does not set trailing and bottom to negative values.

We could potentially rewrite that method to adhere to `UIEdgeInsets` documentation, but I guess we'll break some stuff.

Reply:
`UIEdgeInsets` does seem to be equivalent to `padding` in HTML/CSS, indicating to me that the error lies in how we set up the constraints from them.

Example: https://stackoverflow.com/questions/27459746/adding-space-padding-to-a-uilabel

Reply:
I'll just do this the old fashioned way for now to remove ambiguity.

Reply:
Yes, the problem lies in `fillInSuperview(insets:)`.